stats/view/worker.go

```diff
@@ -290,8 +295,11 @@ func (w *worker) start() {
 func (w *worker) Stop() {
 	prodMgr := metricproducer.GlobalManager()
 	prodMgr.DeleteProducer(w)
-
-	w.quit <- true
+	select {
```

Comment:
Sorry, i've lost context on this.  Why are these (this block, and L289) changes required?  I believe this means you can't call start again after stop, since the channel is closed.

Reply:
That's correct. We can refactor to make it so, but the objective was to enable calls to `Stop()` multiple times.

Reply:
I think we will need to do that.  We can't make breaking changes at this point in the project.

Reply:
Actually, it already didn't work to `Start()` after `Stop()`. `w.c` was closed on quit.